handler/auth/guestaccess/disable.go

```diff
@@ -0,0 +1,65 @@
+/*
+Copyright © 2020 Portworx
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package guestaccess
+
+import (
+	"github.com/portworx/pxc/pkg/cliops"
+	"github.com/portworx/pxc/pkg/commander"
+	"github.com/portworx/pxc/pkg/portworx"
+	"github.com/portworx/pxc/pkg/util"
+	"github.com/spf13/cobra"
+)
+
+var disableGuestAccessCmd *cobra.Command
+
+var _ = commander.RegisterCommandVar(func() {
+	disableGuestAccessCmd = &cobra.Command{
+		Use:   "disable",
+		Short: "Disable guest access role",
+		Example: `
```

Comment:
I talked to Nathan, and there is no need for examples when there is only one way to do it.